tutorials/remote-staking-xcm.md

```diff
@@ -179,6 +242,47 @@ Verify that the structure of your XCM message resembles the below image, then pr
 !!! note
     The encoded call data for the call configured above is `0x630001010100a10f020c00040000010403001300008a5d78456301130000010403001300008a5d784563010006010700902f5009b80c113a7d3048f3cb0391bb44b518e5729f07bcc7a45d000064a7b3b6e00d00000000000000002c01000025000000`.
 
+And that’s it! To verify that your delegation was successful, you can visit [Subscan](https://moonbase.subscan.io/){target=_blank} to check your staking balance. Be advised that it may take a few minutes before your staking balance is visible on Subscan. Additionally, be aware that you will not be able to see this staking operation on Moonscan, because we initiated the delegation action directly via the parachain staking pallet (on the substrate side) rather than through the staking precompile (on the EVM).
+
+## Sending the XCM Instructions via the Polkadot API {: #sending-the-xcm-instructions-via-the-polkadot-api }
+
+In this section we'll be constructing and sending the XCM instructions via the Polkadot API. Just like the above sections, we'll be crafting an XCM message that will transport our remote execution instructions to the Moonbase Alpha parachain to ultimately stake our desired amount of DEV tokens to a chosen collator. After adding the seed phrase of your development account on Moonbase Relay, you can construct and send the transaction via the Polkadot API as follows:
+
+```javascript
+// Import
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+// Construct API provider
+const wsProvider = new WsProvider('wss://frag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network');
+const api = await ApiPromise.create({ provider: wsProvider });
+
+// Import the keyring as required
+import { Keyring } from '@polkadot/api';
+
+// Initialize wallet key pairs
+const keyring = new Keyring({ type: 'sr25519' });
+const otherPair = await keyring.addFromUri('YOUR DEV SEED PHRASE HERE');
+console.log(`Derived Address from Private Key: ${otherPair.address}`);
+
+// Form the transaction
+//const tx = await api.tx.balances.transfer(bob, 1000000000000);
+const dest = { V1: { parents: 0, interior: { X1: { Parachain: 1000 } } } };
+const message = { V2: [{WithdrawAsset:[{id:{concrete:{parents:0,interior:{X1:{PalletInstance: 3}}}},fun: {Fungible: 100000000000000000n}}]},{BuyExecution:[{id:{Concrete:{parents:0,interior:{X1:{PalletInstance: 3}}}},fun: {Fungible: 100000000000000000n}}, {unlimited:null}]},{Transact:{originType: "SovereignAccount",requireWeightAtMost:40000000000n,call:{encoded:'0x0c113a7d3048f3cb0391bb44b518e5729f07bcc7a45d000064a7b3b6e00d00000000000000002c01000025000000'}}}]};
```

Comment:
It might be helpful to add comments above each of these lines explaining what these couple of lines are doing